src/components/Header/index.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+const Header = ({heading, color, size}) => {
+    if (size === 3) {
+        return(
+            <div className="header">
```

Comment:
@kerry-okpere, I'm curious to know why the `Header` component is returning a `div`